docs/mesos/DockerizedDeployment.md

```diff
@@ -1,220 +1 @@
-    sudo docker pull calico/node:v0.8.0
-
-Next, download the unit files
-
-    wget https://github.com/projectcalico/calico-mesos/releases/download/v0.1.2/units.tgz
-    tar -xzf units.tgz
-
-## Calico
-
-`calicoctl` is a small CLI tool to control your Calico network.  It's used to start Calico services on your compute host, as well as inspect and modify Calico configuration.
-
-    curl -L -O https://github.com/projectcalico/calico-containers/releases/download/v0.8.0/calicoctl
-    chmod +x calicoctl
-    sudo cp calicoctl /usr/bin/
-
-You can learn more about `calicoctl` by running `calicoctl --help`.
-
-You'll need to configure Calico with the correct location of the etcd service.  In the following line, replace `<MASTER_IP>` with the IP address of the Master node.
-
-    sudo sh -c 'echo ETCD_AUTHORITY=<MASTER_IP>:4001 > /etc/sysconfig/calico'
-
-Then, enable the Calico service via `systemd`
-
-    sudo cp calico.service /usr/lib/systemd/system/
-    sudo systemctl enable calico.service
-    sudo systemctl start calico.service
-
-Verify Calico is running
-
-    calicoctl status
-
-## Mesos Agent
-
-Use the following commands to tell the Mesos Agent where to find Zookeeper.  The Mesos Agent uses Zookeeper to keep track of the current Mesos Master.  We installed it on the same host as the Mesos Master earlier, so substitute the name or IP of that host for `<ZOOKEEPER_IP>`:
-
-    sudo sh -c 'echo ZK=<ZOKEEPER_IP> > /etc/sysconfig/mesos-agent'
-
-You also need to specify the IP address of the Agent to connect to the Mesos cluster.  Run the following command, replacing `<AGENT_IP>` with the Agent's IP address.
-
-    sudo sh -c 'echo IP=<AGENT_IP> >> /etc/sysconfig/mesos-agent'
-
-Then, enable the Mesos Agent service
-
-    sudo cp mesos-agent.service /usr/lib/systemd/system/
-    sudo systemctl enable mesos-agent.service
-    sudo systemctl start mesos-agent.service
-
-# Test your cluster
-
-To ensure that your cluster is properly networking containers with Calico and enforcing policy as expected, run the Calico Mesos Test Framework, which launches various tasks across your Mesos cluster:
-```
-docker run calico/calico-mesos-framework 172.18.8.101:5050
-```
-> NOTE: Some tests require multiple hosts to ensure cross-host communication, and may fail unless you are running 2+ agents.
-
-Additionally, you can launch your own tasks using Marathon. See our [Marathon Task Launch Instructions](README.md#3-launching-tasks) for more information.
-
-[calico]: http://projectcalico.org
-[mesos]: https://mesos.apache.org/
-[net-modules]: https://github.com/mesosphere/net-modules
-[![Analytics](https://ga-beacon.appspot.com/UA-52125893-3/calico-containers/docs/mesos/DockerizedDeployment.md?pixel)](https://github.com/igrigorik/ga-beacon)
+The contents you are looking for have moved to the new calico-mesos-deployments repo. [Follow this link to view them](https://github.com/projectcalico/calico-mesos-deployments/blob/master/docs/DockerizedDeployment.md)
```

Comment:
My thought was that I only want them clicking on the link that takes them directly to the doc they were trying to look at originally